app/directives/formats/views/form-loader.html

```diff
@@ -1,5 +1,29 @@
 ﻿<div ng-cloak style="background:#fff">
 	<div ng-include="'/app/directives/formats/views/print-header.html'" ng-if="printMode"></div>
+	<!-- <link href="/app/css/print-friendly.css" rel="stylesheet"  ng-if="printMode"> -->
+	<style  ng-if="printMode">
```

Comment:
can use std html ```<style media="print">```

Reply:
to make the pdf look like the online form i use printmode screen so then the media css won't be applied.